### DISCUSSION TITLE
28978 Ixion missing

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3211] [General]
Ixion missing in Stellarium. It is a celestial object of the solar system.

https://en.wikipedia.org/wiki/28978_Ixion

**Comment by alex-w** [comment]
Which reasons to add Ixion into default list of minor bodies?

**Comment by gzotti** [comment]
There are about 28950 other bodies which are not included before this number. Please use the Solar System Editor to add what you need (User Guide, ch.13.7). Don't overdo it, with thousands of minor bodies Stellarium gets slow.

**Comment by ilkant** [comment]
Thanks for advice. There are now a new study to mee: MPC format. Ixion was example because it is on Wikipedia.
